Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+def initializeEnvironment() {
+
+    env.DRIVER_DISPLAY_NAME = 'Cassandra Ruby Driver'
+    env.DRIVER_METRIC_TYPE = 'oss'
+
+    env.GIT_SHA = "${env.GIT_COMMIT.take(7)}"
+    env.GITHUB_PROJECT_URL = "https://${GIT_URL.replaceFirst(/(git@|http:\/\/|https:\/\/)/, '').replace(':', '/').replace('.git', '')}"
+    env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
+    env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
+}
+
+def installDependencies() {}
+
+def buildDriver() {}
+
+def executeTests() {
+    sh label: 'Execute all tests', script: '''#!/bin/bash -le
+        rake test
+    '''
+}
+
+pipeline {
+    agent none
+
+    // Global pipeline timeout
+    options {
+	timeout(time: 10, unit: 'HOURS')
+	buildDiscarder(logRotator(artifactNumToKeepStr: '10', // Keep only the last 10 artifacts
+				  numToKeepStr: '50'))        // Keep only the last 50 build records
+    }
+
+    environment {
+	CCM_ENVIRONMENT_SHELL = '/usr/local/bin/ccm_environment.sh'
+    }
+
+    stages {
+	stage('Per-Commit') {
+	    environment {
+		OS_VERSION = 'ubuntu/bionic64/ruby-driver'
+	    }
+
+	    matrix {
+		axes {
+		    axis {
+			name 'CASSANDRA_VERSION'
+			values '2.1', '3.11', '4.0'
+		    }
+		    axis {
+			name 'RUBY_VERSION'
+			values '2.3.6', '2.4.3', '2.7.0', 'jruby-9.1.15.0'
+		    }
+		}
+		agent {
+		    label "${OS_VERSION}"
+		}
+
+		stages {
+		    stage('Initialize-Environment') {
+			steps {
+			    initializeEnvironment()
+			    /*script {
+				if (env.BUILD_STATED_SLACK_NOTIFIED != 'true') {
+				    notifySlack()
+				}
+			    }*/
+			}
+		    }
+		    stage('Describe-Build') {
+			steps {
+			    describePerCommitStage()
```

Comment:
I think the builds are failing at this stage, likely because this function is undefined?

Reply:
Yeah, this was a copy/paste error on my part.  Working on fixing now.